configuration/config-txt/overclocking.md

```diff
@@ -10,30 +10,61 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
+| over_voltage_sdram_p | SDRAM phy voltage adjustment. [-16,8] equates to [0.8V,1.4V] with 0.025V steps. |
 | force_turbo | Forces turbo mode frequencies even when the ARM cores are not busy. Enabling this may set the warranty bit if `over_voltage_*` is also set. |
-| initial_turbo | Enables turbo mode from boot for the given value in seconds, or until cpufreq sets a frequency. For more information [see here](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=6201&start=425#p180099). The default value is `0`, maximum value is `60`. |
-| arm_freq_min | Minimum value of arm_freq used for dynamic frequency clocking. The default value is `700` for Pi Zero/Pi 1, `600` for Pi 2/Pi 3. |
-| core_freq_min | Minimum value of `core_freq` used for dynamic frequency clocking. The default value is `250`. On Pi 4B the default is`275` when `hdmi_enable_4kp60` is set|
-| gpu_freq_min | Minimum value of `gpu_freq` used for dynamic frequency clocking. The default value is `250`, or `500` on Pi 4B. |
-| h264_freq_min | Minimum value of `h264_freq` used for dynamic frequency clocking. The default value is `250`, or `500` on Pi 4B. |
-| isp_freq_min | Minimum value of `isp_freq` used for dynamic frequency clocking. The default value is `250`, or `500` on Pi 4B. |
-| v3d_freq_min | Minimum value of `v3d_freq` used for dynamic frequency clocking. The default value is `250`, or `500` on Pi 4B. |
-| sdram_freq_min | Minimum value of `sdram_freq` used for dynamic frequency clocking. The default value is `400`. |
-| over_voltage_min | Minimum value of `over_voltage` used for dynamic frequency clocking. The default value is `0`. |
-| temp_limit | Overheat protection. This sets the clocks and voltages to default when the SoC reaches this value in Celsius.  The default value is `85`. Values over 85 are clamped to 85.|
-| temp_soft_limit | **3A+/3B+ only**. CPU speed throttle control. This sets the temperature at which the CPU clock speed throttling system activates. At this temperature, the clock speed is reduced from 1400Mhz to 1200Mhz.  Defaults to `60`, can be raised to a maximum of `70`, but this may cause instability.
+| initial_turbo | Enables turbo mode from boot for the given value in seconds, or until cpufreq sets a frequency. For more information [see here](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=6201&start=425#p180099). The maximum value is `60`. |
+| arm_freq_min | Minimum value of arm_freq used for dynamic frequency clocking. |
+| core_freq_min | Minimum value of `core_freq` used for dynamic frequency clocking. On Pi 4B the default is`275` when `hdmi_enable_4kp60` is set|
+| gpu_freq_min | Minimum value of `gpu_freq` used for dynamic frequency clocking.|
+| h264_freq_min | Minimum value of `h264_freq` used for dynamic frequency clocking.|
+| isp_freq_min | Minimum value of `isp_freq` used for dynamic frequency clocking.|
+| v3d_freq_min | Minimum value of `v3d_freq` used for dynamic frequency clocking.|
```

Comment:
@popcornmix Sorry for being pedantic, but is there also a `hevc_freq_min` setting on the Pi4?